tests/build/builder/test_docker.py

```diff
@@ -4,42 +4,62 @@
 import docker.errors
 import pytest
 import tempfile
+from testcontainers.core.container import DockerContainer
+
 from ebonite.build.builder.base import use_local_installation
-from ebonite.build.builder.docker_builder import _DockerfileGenerator, DockerBuilder, create_docker_client
-from ebonite.build.docker_objects import DockerImage
+from ebonite.build.builder.docker import _DockerfileGenerator, DockerBuilder
+from ebonite.build.docker import DockerImage, RemoteDockerRegistry, create_docker_client
 
 from tests.build.conftest import has_docker
 from tests.build.builder.test_base import ProviderMock, SECRET
 
 CLEAN = True
 IMAGE_NAME = 'ebonite_test_docker_builder_image'
 
+REGISTRY_PORT = 5000
+REGISTRY_HOST = f'localhost:{REGISTRY_PORT}'
 
 no_docker = pytest.mark.skipif(not has_docker(), reason='docker is unavailable or skipped')
 
 
 @pytest.fixture
-def docker_builder():
+def docker_builder_local_registry():
     with use_local_installation():
         yield DockerBuilder(ProviderMock(), DockerImage(IMAGE_NAME))
 
 
+@pytest.fixture
+def docker_builder_remote_registry():
+    with use_local_installation(), DockerContainer('registry:latest').with_bind_ports(REGISTRY_PORT, REGISTRY_PORT):
+        yield DockerBuilder(ProviderMock(),
+                            DockerImage(IMAGE_NAME, registry=RemoteDockerRegistry(REGISTRY_HOST)))
```

Comment:
REGISTRY_PORT is not used?

Reply:
REGISTRY_HOST contains REGISTRY_PORT, see above